core/src/apps/management/wipe_device.py

```diff
@@ -1,12 +1,12 @@
 import storage
 from trezor.messages.Success import Success
-from trezor.ui.layouts import confirm_wipe, require
+from trezor.ui.layouts import confirm_wipe
 
 from .apply_settings import reload_settings_from_storage
 
 
 async def wipe_device(ctx, msg):
-    await require(confirm_wipe(ctx))
+    await confirm_wipe(ctx)
```

Comment:
There's some styling missing but since `confirm_action` has everything (w/ kitchen sink) anyway I fixed this in 28f676ca68135ca4a6b46cdf7b18815a361e4b36.